core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -40,6 +40,13 @@
         </schema-based-editor>
       </div>
     </div>
+    <div class="oppia-translation-tips-container">
```

Comment:
What does this look like for a language that does not have translation tips? Can you post a screenshot? Ideally, we would hide the entire container in that case.

Reply:
After moving the `ng-if="TRANSLATION_TIPS[activeLanguageCode]" `to the `<div>` It shows nothing now. Posted a screenshot

Reply:
Catalan Example
<img width="649" alt="Screen Shot 2021-04-01 at 9 32 50 AM" src="https://user-images.githubusercontent.com/35385366/113325411-3d571400-92cd-11eb-8bef-5857a042be5d.png">